### ISSUE TITLE
Buttons missing

### ISSUE THREAD

**Issue by DarkXLord20** [SachinVin/citra_android#150]
Some buttons like a b x y are missing in latest build

**Comment by SachinVin** [comment]
rotate to landscape

**Comment by EnergyCube** [comment]
@SachinVin Yeah, but I had quite a few people on the PlayStore who was complaining about that. Why we don't add a layout scheme for portrait ?
